### PR TITLE
Add spring boot actuator mvc endpoint to turn on/off toggles UNTESTED

### PR DESCRIPTION
**Pull request by borislopezaraoz**
Existing endpoint:
GET /togglz -> all toggles statuses

This PR adds new endpoints:

GET /togglz/`<TOGGLE_NAME>` -> one toggle status
POST /togglz/`<TOGGLE_NAME>` -> set one toggle status (with body)
example with curl:
```
curl -H "Content-Type: application/json" -X POST -d '{"enabled":"true"}' https://example.com/app/path/togglz/<TOGGLE_NAME>
```

TODO: add unit tests
TODO: be able to also update strategy & params

I used as template spring boot loggers (mvc) endpoint.
org.springframework.boot.actuate.endpoint.LoggersEndpoint
org.springframework.boot.actuate.endpoint.mvc.LoggersMvcEndpoint

This code is working in my app and it is passing togglz existing tests in TravisCI, but I couldn't run togglz unit tests locally (surefire errors), so I didn't add a new test, sorry about that.

### CODE REVIEW

File: spring-boot-starter/src/main/java/org/togglz/spring/boot/autoconfigure/TogglzEndpoint.java [hunk 1]
```diff
@@ -54,6 +54,17 @@ public TogglzEndpoint(FeatureManager featureManager) {
         return features;
     }
 
+    public Optional<Feature> get(String name) {
+        return featureManager.getFeatures().stream().filter(f -> name.equals(f.name())).findFirst();
```
Comment (hexmind) [inline spring-boot-starter/src/main/java/org/togglz/spring/boot/autoconfigure/TogglzEndpoint.java]: TogglzEndpoint.java:[58,63] lambda expressions are not supported in -source 1.7
https://travis-ci.org/togglz/togglz/jobs/228788533

File: spring-boot-starter/src/main/java/org/togglz/spring/boot/autoconfigure/ActuatorPostMapping.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ * {@code application/vnd.spring-boot.actuator.v1+json} requests and produce
+ * {@code application/json} or {@code application/vnd.spring-boot.actuator.v1+json}
+ * responses.
+ *
+ * @author Andy Wilkinson
+ */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@RequestMapping(method = RequestMethod.POST, consumes = {
```
Comment (hexmind) [inline spring-boot-starter/src/main/java/org/togglz/spring/boot/autoconfigure/ActuatorPostMapping.java]: Can you brak it one attribute per line (outer annotation)?

**Comment by borislopezaraoz** [comment]
This PR adds new endpoints.
Existing endpoint:
GET /togglz -> all toggles statuses
GET /togglz/<TOGGLE NAME> -> toggle status
POST /togglz/<TOGGLE NAME> -> set toggle status (with body)
example with curl:
```
curl -H "Content-Type: application/json" -X POST -d '{"enabled":"true"}' https://example.com/app/path/togglz/<toggle name>
```

**Comment by chkal** [comment]
Thanks a lot the pull request. Having the ability to modify features state would be awesome.

However, I really thing those remaining TODO items needs to be addressed before we can merge this. Especially the support for strategies and their params.

**Comment by ruifigueira** [comment]
This seems a very useful functionality, but unfortunately `@EndpointMvcAdapter` is no longer supported in Spring Boot 2.

`@Endpoint` supports `@WriteOperation` methods, I'll try to do a simple PoC to check if we can do the same in Spring Boot 2.

**Comment by bennetelli** [comment]
It's possible to turn on and off togglz via plain Spring Boot actuator endpoint already. So I would like to close this PR.

**Pull request closed without merging**